git_machete/cmd.py

```diff
@@ -424,6 +425,16 @@ def get_git_dir():
     return git_dir
 
 
+def get_abs_git_dir():
+    global abs_git_dir
+    if not abs_git_dir:
+        try:
+            abs_git_dir = popen_git("rev-parse", "--absolute-git-dir").strip()
```

Comment:
What is the minimum version of git that supports this flag?...

Reply:
It seems that minimum version is `2.13.0`. I will try to implement it differently as we want to support `2.7` version as well.